ArduCopter/mode_systemid.cpp

```diff
@@ -81,14 +81,53 @@ bool ModeSystemId::init(bool ignore_checks)
     }
 
     // if landed and the mode we're switching from does not have manual throttle and the throttle stick is too high
-    if (motors->armed() && copter.ap.land_complete && !copter.flightmode->has_manual_throttle()) {
+    if (motors->armed() && copter.ap.land_complete && (!copter.flightmode->has_manual_throttle() || copter.flightmode->mode_number() == Mode::Number::LOITER)) {
```

Comment:
I don't think the extra loiter check here is doing anything? `has_manual_throttle` returns false for loiter. So `!copter.flightmode->has_manual_throttle()` will already be true so there is no need to `|| true`?

Reply:
Good catch.  This must be a left over from early attempts to keep it from switching into system ID for axes 1-13 while in loiter.  I have captured that behavior in the sections below.

This really needs to capture that a user can't switch into system ID if they aren't flying.  For heli's that can be quite dangerous and I don't think multicopters want to takeoff with the aircraft doing sysid.